File: examples-sdk/zig/601-storage-s3-basic/main.zig
// 601-storage-s3-basic — Zig SDK equivalent
// Compile: vil compile --from zig --input 601-storage-s3-basic/main.zig --release

const vil = @import("vil");

pub fn main() void {
    var server = vil.Server.init("app", 8080);
    server.compile();
}

File: examples-sdk/go/601-storage-s3-basic/main.go
// 601-storage-s3-basic — Go SDK equivalent
// Compile: vil compile --from go --input 601-storage-s3-basic/main.go --release
package main

import vil "github.com/OceanOS-id/vil-go"

func main() {
	s := vil.NewServer("app", 8080)
	s.Compile()
}
